src/agent/sign.rs

```diff
@@ -248,10 +249,25 @@ pub async fn sign_request(
     let base64_data = base64::encode(data);
     let relay_id = base64::encode_config(randombytes(32), base64::URL_SAFE);
 
+    let request_id = match proxy {
+        None => None,
+        Some(proxy) => Some(format!("{}@{}", name, proxy.host.clone())),
+    };
+    let auto_accept_token = match request_id.clone() {
+        None => None,
+        Some(request_id) => get_auto_accept("ssh".to_string(), request_id.clone()),
+    };
+
     let mut payload = HashMap::new();
     payload.insert("type", "ssh".to_string());
     payload.insert("data", base64_data);
     payload.insert("userName", name);
+    match auto_accept_token {
```

Comment:
```suggestion
    if let Some(token) = auto_accept_token
```

No else required then. Makes the code a bit more readable.